examples/async/webpack.config.js

```diff
@@ -15,7 +15,6 @@ module.exports = {
   plugins: [
     new webpack.optimize.OccurenceOrderPlugin(),
     new webpack.HotModuleReplacementPlugin(),
-    new webpack.NoErrorsPlugin()
```

Comment:
No, this is not the reason. Redbox only catches runtime errors, it doesn’t help with syntax errors.
NoErrorsPlugin has upsides and downsides. I’m likely to add it back. Please see https://github.com/gaearon/react-transform-boilerplate/issues/122 for a list of its upsides and downsides.